### PR TITLE
AM-3134 Create new role for SSCS post-hearing-judge 

### PR DESCRIPTION
**Pull request by mikebrownccd**
https://tools.hmcts.net/jira/browse/AM-3134
Create new role for SSCS post-hearing-salaried-judge

> **NB: This PR is Dependent  on RAS hmcts/am-role-assignment-service#2159.**
> ~**NB: This PR has temporary changes to Jenkins and Chart file configuration that need reversing before merger.**~

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] Does this PR introduce a breaking change

### CODE REVIEW

**Review by prashanthkumarnb — APPROVED** [review]

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -75,7 +75,8 @@ env.PACT_BROKER_FULL_URL = "https://pact-broker.platform.hmcts.net"
 env.AMQP_HOST = "rd-servicebus-aat.servicebus.windows.net"
 env.CRD_TOPIC_NAME = "rd-caseworker-topic-aat"
 env.JRD_TOPIC_NAME = "rd-judicial-topic-aat"
-env.ROLE_ASSIGNMENT_URL = "http://am-role-assignment-service-aat.service.core-compute-aat.internal"
+// AM-3134 env.ROLE_ASSIGNMENT_URL = "http://am-role-assignment-service-aat.service.core-compute-aat.internal"
+env.ROLE_ASSIGNMENT_URL = "http://am-role-assignment-service-pr-2159.preview.platform.hmcts.net"
```
Comment (mattnayler) [inline Jenkinsfile_CNP:79]: NB: This temporary change must be reverted before merging.

**Review by mattnayler — APPROVED** [review]
LGTM

**Review by mattnayler — APPROVED** [review]
Re-approved now that FTAs are green.

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -75,7 +75,8 @@ env.PACT_BROKER_FULL_URL = "https://pact-broker.platform.hmcts.net"
 env.AMQP_HOST = "rd-servicebus-aat.servicebus.windows.net"
 env.CRD_TOPIC_NAME = "rd-caseworker-topic-aat"
 env.JRD_TOPIC_NAME = "rd-judicial-topic-aat"
-env.ROLE_ASSIGNMENT_URL = "http://am-role-assignment-service-aat.service.core-compute-aat.internal"
+// AM-3134 env.ROLE_ASSIGNMENT_URL = "http://am-role-assignment-service-aat.service.core-compute-aat.internal"
+env.ROLE_ASSIGNMENT_URL = "https://am-role-assignment-service-pr-2159.preview.platform.hmcts.net"
```
Comment (mattnayler) [inline Jenkinsfile_CNP:79]: NB: This temporary change must be reverted before merging.

File: charts/am-org-role-mapping-service/values.preview.template.yaml [hunk 3]
```diff
@@ -34,7 +34,8 @@ java:
   environment:
     IDAM_S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
-    ROLE_ASSIGNMENT_APP_URL: http://am-role-assignment-service-aat.service.core-compute-aat.internal
+    # AM-2159 ROLE_ASSIGNMENT_APP_URL: http://am-role-assignment-service-aat.service.core-compute-aat.internal
+    ROLE_ASSIGNMENT_APP_URL: https://am-role-assignment-service-pr-2159.preview.platform.hmcts.net
```
Comment (mattnayler) [inline charts/am-org-role-mapping-service/values.preview.template.yaml:38]: NB: This temporary change must be reverted before merging.

**Review by mattnayler — APPROVED** [review]
Re-approved after 58761c62b13aa77c245d529cb6d5e32c0fd33165.

File: charts/am-org-role-mapping-service/values.preview.template.yaml (hunk 3 above)
Comment (mattnayler) [reply charts/am-org-role-mapping-service/values.preview.template.yaml]: Resolved in 58761c62b13aa77c245d529cb6d5e32c0fd33165.

**Pull request merged**